src/translib/intf_app.go

```diff
@@ -342,6 +337,31 @@ func (app *IntfApp) processGet(dbs [db.MaxDB]*db.DB) (GetResponse, error) {
 				if err != nil {
 					return GetResponse{Payload: payload, ErrSrc: AppErr}, err
 				}
+
```

Comment:
General comment : Can we have get for individual state and counter attributes to 2 separate functions? Looks like, function is getting bigger. What do you think? We can take this as part of cleanup too.

Reply:
We'll take this one in the next check-in

Reply:
Sure, no problem.